components/search/SearchSelectedFacets.vue

```diff
@@ -7,7 +7,12 @@
       class="mr-2"
       data-qa="filter badge"
     >
-      {{ $t('formatting.labelledValue', { label: $t(`facets.${selectedFacet.facetName}`), value: selectedFacet.fieldValue}) }}
+      <template v-if="selectedFacet.facetName === 'THEME'">
```

Comment:
Let's make this more reusable by switching to:
```js
<template v-if="$te(`facets.${selectedFacet.facetName}.options`)">
```

Reply:
Good suggestion, thanks. I have updated the code. 

I might add a test with options as well, and update the mocked $te.